src/compiler/crystal/tools/doc/html/type.html

```diff
@@ -19,7 +19,9 @@ <h1 class="type-name">
 <% if type.program? %>
   <%= type.full_name.gsub("::", "::<wbr>") %>
 <% else %>
-  <span class="kind"><%= type.abstract? ? "abstract " : ""%><%= type.kind %></span> <%= type.full_name.gsub("::", "::<wbr>") %>
+  <span class="kind">
+    <%= type.abstract? ? "abstract " : ""%><%= type.visibility %><%= type.kind %>
```

Comment:
Aren't we missing spaces around the visibility, for the case where there a visibility + kind?